modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +62,20 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+. To keep the PVCs for reuse with other pods, keep the labels or PVC names that you need to reclaim the PVCs.
+
+. Optional: If you do not want to reclaim the PVCs, you can delete them.
++
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
++
+If the value of the PV `persistentVolumeReclaimPolicy` property is `delete`, deleting the PVC also deletes the corresponding PV. This action releases the backing storage and removes all data, such as collected log data.
+
 
 .. Switch to the *Storage* -> *Persistent Volume Claims* page.
 
 .. Click the Options menu {kebab} next to each PVC and select *Delete Persistent Volume Claim*.
 
+.. If the value of the PV `persistentVolumeReclaimPolicy` property is `retain`, use `$ oc delete <pv-name>` to delete the PVs.
```

Comment:
@rolfedh thank you for your updates. Is it possible to delete the PVs through the UI? The procedure relates to the UI until this step, which then switches to using the CLI.

If it is possible to do this through the UI, you could update this step to provide the UI method instead, for consistency.

If the step is not possible through the UI:

* You could move the command into a code block, so that the reader can use the copy button and for scannability. If you do decide to leave it as part of the sentence, please remove the `$` prompt.
* Please change `<pv-name>` to `<pv_name>`.
* You could state in the step that this command needs to be issued through the CLI, to clarify why the procedure switches from the UI for the last step.

Reply:
AFAIK there isn't a UI method. I've replaced the step+command for removing the PV with instructions to go see a more comprehensive topic on the same. I debated with how to phrase this, as it is, or in an admonition. What are your thoughts?

Reply:
@rolfedh my preference would be to include the actual step in the procedure, so that it is complete in itself, rather than referring to the "Additional resources" section for that. As per my previous update, you could state that the step is issued through the CLI, to clarify that for the reader.

I also think that "Optional:" is not required in this bullet point, since the whole of step 6 is optional already.